source/features/toggle-files-button.tsx

```diff
@@ -33,17 +33,26 @@ function addButton(): void {
 async function toggleHandler(): Promise<void> {
 	const isHidden = select('.repository-content')!.classList.toggle('rgh-files-hidden');
 	await (isHidden ? cache.set(cacheKey, true) : cache.delete(cacheKey));
+
+	// Remove notice after the first click
+	select('.rgh-files-hidden-notice')?.remove();
 }
 
 async function init(): Promise<Deinit[]> {
 	const repoContent = (await elementReady('.repository-content'))!;
+
 	if (await cache.get<boolean>(cacheKey)) {
 		repoContent.classList.add('rgh-files-hidden');
+		select('#files')!.after(
+			<div className="py-1 text-right text-small rgh-files-hidden-notice" style={{paddingRight: '19px'}}>
```

Comment:
Why 19 pixels exactly? Could this be replaced e.g. by `.pr-3` (which would be 16px)?

Reply:
```suggestion
			// 19px align this icon with the UnfoldIcon above it
			<div
				className="py-1 text-right text-small rgh-files-hidden-notice"
				style={{paddingRight: '19px'}}
			>
```